### PR TITLE
Use orbs on CircleCI config and bump Scala versions

### PR DESCRIPTION
**Pull request by josemiguelmelo**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,171 +1,49 @@
...
-      - checkout_and_version
-      - sbt_dependencies:
+      - codacy/checkout_and_version:
+          write_sbt_version: true
+      - codacy/sbt:
+          name: publish_docker_local
+          cmd: |
+            sbt "set scalafmtUseIvy in ThisBuild := false;
+                 test;
+                 set name := \"$CIRCLE_PROJECT_REPONAME\";
```
Comment (lolgab) [inline .circleci/config.yml:20]: can you add scalafmt checks here? Remember to run scalafmt to let they pass

File: build.sbt [hunk 2]
```diff
@@ -4,18 +4,18 @@ name := """codacy-engine-phpmd"""
 
 version := "1.0-SNAPSHOT"
 
-scalaVersion := "2.12.7"
+scalaVersion := "2.13.1"
 
 resolvers ++= Seq(
   "Sonatype OSS Snapshots" at "https://oss.sonatype.org/content/repositories/releases",
   "Typesafe Repo" at "http://repo.typesafe.com/typesafe/releases/"
 )
```
Comment (lolgab) [inline build.sbt]: This section is not needed

File: build.sbt [hunk 3]
```diff
@@ -37,15 +37,16 @@ val installAll =
...
-  for {
-    path <- (src ***).get
-    if !path.isDirectory
-  } yield path -> path.toString.replaceFirst(src.toString, dest)
-}
+    val docFiles = for {
+      path <- src.allPaths.get if !path.isDirectory
+    } yield path -> path.toString.replaceFirst(src.toString, dest)
+
+    docFiles
```
Comment (lolgab) [inline build.sbt]: Here you can return directly without creating an intermediate variable

File: build.sbt [hunk 4]
```diff
@@ -57,13 +58,13 @@ daemonGroup in Docker := dockerGroup
...
-    Cmd("RUN", s"adduser -u 2004 -D $dockerUser"),
-    ExecCmd("RUN", Seq("chown", "-R", s"$dockerUser:$dockerGroup", "/docs"): _*)
-  )
+  case cmd @ Cmd("ADD", _) =>
+    List(
+      Cmd("RUN", s"adduser -u 2004 -D $dockerUser"),
+      cmd,
+      Cmd("RUN", installAll),
+      Cmd("RUN", "mv /opt/docker/docs /docs"),
+      ExecCmd("RUN", "chown", "-R", s"$dockerUser:$dockerUser", "/docs")
```
Comment (lolgab) [inline build.sbt]: Here you can use Cmd and a single string as in the command above

File: project/build.properties [hunk 5]
```diff
@@ -1,4 +1,4 @@
 #Activator-generated Properties
 #Tue Jun 23 14:00:32 EEST 2015
-template.uuid=7faf8e1e-4e8d-4387-8159-642b50383096
-sbt.version=0.13.8
+#template.uuid=7faf8e1e-4e8d-4387-8159-642b50383096
```
Comment (lolgab) [inline project/build.properties]: You can remove these comments here

**Review by lolgab — CHANGES_REQUESTED** [review]

File: build.sbt (hunk 3 above)
Comment (josemiguelmelo) [reply build.sbt]: Done ✅

File: build.sbt (hunk 2 above)
Comment (josemiguelmelo) [reply build.sbt]: Done ✅

File: .circleci/config.yml (hunk 1 above)
Comment (josemiguelmelo) [reply .circleci/config.yml:20]: Done ✅

File: build.sbt (hunk 4 above)
Comment (josemiguelmelo) [reply build.sbt]: Done ✅

File: project/build.properties (hunk 5 above)
Comment (josemiguelmelo) [reply project/build.properties]: Done ✅

**Review by lolgab — APPROVED** [review]

**Pull request merged**